Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -1781,20 +1782,23 @@ static HRESULT _CreatePatternBrush(
 
     size_t bitsPerComponent = 8;
     size_t bytesPerRow = 4 * tileSize.size.width;
-    CGBitmapInfo bitmapInfo = kCGImageAlphaPremultipliedFirst | kCGBitmapByteOrder32Big;
+    CGBitmapInfo bitmapInfo = kCGImageAlphaPremultipliedFirst | kCGBitmapByteOrderDefault;
 
     woc::unique_cf<CGContextRef> patternContext{ CGBitmapContextCreate(
         nullptr, tileSize.size.width, tileSize.size.height, bitsPerComponent, bytesPerRow, colorspace.get(), bitmapInfo) };
     RETURN_HR_IF_NULL(E_UNEXPECTED, patternContext);
 
+    // Determine if this pattern is a colored pattern (the coloring is specified in the pattern callback) or if it is stencil pattern (the
+    // color is set outside and not within the pattern callback)
+    bool isColored = _CGPatternIsColored(pattern);
+
     // Stage the drawing context
-    RETURN_IF_FAILED(std::forward<ContextStageLambda>(contextStage)(patternContext.get()));
+    RETURN_IF_FAILED(std::forward<ContextStageLambda>(contextStage)(patternContext.get(), isColored));
```

Comment:
yes but this helps make this more general.
Leaving it as it is.

---
In reply to: [101356532](https://github.com/Microsoft/WinObjC/pull/1952#discussion_r101356532) [](ancestors = 101356532)